goreleaser.yml

```diff
@@ -117,12 +115,10 @@ dockers:
     extra_files:
     - contrib/
   - image_templates:
-      - "docker.io/aquasec/trivy:{{ .Version }}-arm64"
-      - "docker.io/aquasec/trivy:latest-arm64"
-      - "ghcr.io/aquasecurity/trivy:{{ .Version }}-arm64"
-      - "ghcr.io/aquasecurity/trivy:latest-arm64"
-      - "public.ecr.aws/aquasecurity/trivy:latest-arm64"
-      - "public.ecr.aws/aquasecurity/trivy:{{ .Version }}-arm64"
+      - "docker.io/devopps/trivy:{{ .Version }}-arm64"
```

Comment:
devops?

Reply:
so sorry about that, I forgot to revert the changes that I've been made for testing purposes.